out/test/test/input.txt

```diff
@@ -0,0 +1,163 @@
+##########################################################
+# invalid command
+##########################################################
+
+  # should recognise invalid command
+  sfdfd
+
+##########################################################
+# commands using list index before any listing created
+##########################################################
+
+  # should show appropriate message when no listing has happened yet
+  delete 1
+  view 1
+  viewall 1
+
+##########################################################
+# clean and check state
+##########################################################
+
+  # address book should be emptied
+  clear
+  list
+
+##########################################################
+# test add person command, setup state for further tests
+##########################################################
+
+  # should catch invalid args format
+  add wrong args wrong args
+  add Valid Name 12345 e/valid@email.butNoPhonePrefix a/valid, address
+  add Valid Name p/12345 valid@email.butNoPrefix a/valid, address
+  add Valid Name p/12345 e/valid@email.butNoAddressPrefix valid, address
+  add Valid Name p/12345 e/valid@email.butNoTagPrefix a/valid, address t/goodTag noPrefixTag
+  
+  # should catch invalid person data
+  add []\[;] p/12345 e/valid@e.mail a/valid, address
+  add Valid Name p/not_numbers e/valid@e.mail a/valid, address
+  add Valid Name p/12345 e/notAnEmail a/valid, address
+  add Valid Name p/12345 e/valid@e.mail a/valid, address t/invalid_-[.tag
+
+  # should add correctly and list non private information
+  add Adam Brown p/111111 e/adam@gmail.com a/111, alpha street
+  list
+  add Betsy Choo pp/222222 pe/benchoo@nus.edu.sg pa/222, beta street t/secretive
+  list
+  add Charlie Dickson pp/333333 e/charlie.d@nus.edu.sg a/333, gamma street t/friends t/school
+  list
+  add Dickson Ee p/444444 pe/dickson@nus.edu.sg a/444, delta street t/friends
+  list
+  add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
+  list
+
+  # should not allow adding duplicate persons
+  add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
+
+##########################################################
+# test sort command
```

Comment:
good job at adding sort test (: however, ^ avoid staging irrelevant changes in your commit such as changes to add/find command tests. (make a clean pr)